src/Imgix/UrlBuilder.cs

```diff
@@ -166,14 +166,14 @@ private String
             GenerateTargetWidths(
                 double start = 100, double stop = 8192, double tol = 8)
         {
-            List<int> resolutions = new List<int>();
             int MAX_SIZE = 8192;
 
             if (start == stop)
```

Comment:
This PR has the `start == stop` functionality we discussed, I just also wanted to define resolutions right before it's needed (e.g. if we're producing a singular width, no need to allocate the array twice and no need to alloc and then push/add onto etc.).